<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cipher</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Keyword:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="keyword_lineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="decode_pushButton">
          <property name="text">
           <string>Decode</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="encode_pushButton">
          <property name="text">
           <string>Encode</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Ciphertext:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="cipher_textEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Plaintext:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plain_textEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_Close"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_Close</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
